lua/vgui/gnbutton.lua

```diff
@@ -1,3 +1,24 @@
+--- @title:
+--- 	GNButton: <VGUI> Create a button
+--- @note:
+--- 	Parent: GNPanel
+--- @params:
+--- 	GNButton/SetPos( Position ): <function> Set the position of the button
+--- 	GNButton/SetSize( Size ): <function> Set the size of the button
+--- 	GNButton/SetText( Text ): <function> Set the text of the button
+--- 	GNButton/SetFont( Font name ): <function> Add the font name of the button
+--- 	GNButton/SetTextColor( Color ): <function> Add the text color of the button
+--- 	GNButton/SetColor( Color ): <function> Set the color of the button
+--- 	GNButton/SetHoveredColor( Color ): <function> Set the color of the button hovered
+--- 	GNButton/SetClickedColor( Color ): <function> Set the color of the button clicked
+--- 	GNButton/SetHoveredTextColor( Color ): <function> Set the color of the button text hovered
+--- 	GNButton/SetClickedTextColor( Color ): <function> Set the color of the button text clicked
+--- 	GNButton/SetHideLeft( Bool ): <function> Add the hide left of the button
+--- 	GNButton/SetHideRight( Bool ): <function> Add the hide right of the button
```

Comment:
The functions parameters should be on the following form : `<type> <name>`. For example, on `SetColor`, it should be `SetColor( Color color )`. Also, `Bool` should be `Boolean`.